float64 x
float64 y
int32 age
int32 id
